netZooPy/panda/panda.py

```diff
@@ -492,8 +491,11 @@ def processData(
 
         with Timer("Creating motif network ..."):
             self.motif_matrix_unnormalized = np.zeros((self.num_tfs, self.num_genes))
-            idx_tfs = [tf2idx.get(x, 0) for x in self.motif_data[0]]
-            idx_genes = [gene2idx.get(x, 0) for x in self.motif_data[1]]
+            idx_tfs = [tf2idx.get(x, np.nan) for x in self.motif_data[0]]
```

Comment:
@michelegentili93 there were a few more bugs like the one you reported, can you please have a look and see if this makes sense?